main.py

```diff
@@ -1,15 +1,45 @@
 import datetime
+import os
+import time
 
-def obter_limite():
-    cargo_atual = str(input("Para fazermos sua analise de credito, "
-                            "precisamos que voce entre com algumas informacoes:\nCargo atual: "))
-    salario_atual = float(input("Salario Atual: "))
-    ano_de_nascimento = int(input("Ano de nascimento: "))
-    ano_vigente = datetime.date.today()
-    idade_aproximada = (ano_vigente.year - ano_de_nascimento)
+
+NOME_COMPLETO = "Rodiney Branta"
+
+
+def obter_limite(salario_atual, idade_aproximada):
     valor_credito_disponivel = ((salario_atual*(idade_aproximada / 1000)) +100) 
-    print(f"Seu cargo atual: {cargo_atual} \nSeu Salario atual: {salario_atual} \nSeu ano de nascimento: {ano_de_nascimento}")
-    print(f"Voce tem aproximadamente {idade_aproximada} anos.")
-    print(f"Voce possui o valor em credito de R$ {valor_credito_disponivel:.2f}, disponiveis para compras em nossa loja.")
+    return valor_credito_disponivel
```

Comment:
Eu retiraria a variável, não tem pq ela existir

```suggestion
    return ((salario_atual*(idade_aproximada / 1000)) +100) 
```
e removeria a linha de cima, claro.

Reply:
Olhando o fonte, ficou estranho, pois o nome da função é `obter_limite`, mas a variável com o valor a ser retornado se chama `valor_credito_disponivel`. Meio confuso isso. Estamos aí calculando o que?

Reply:
> Olhando o fonte, ficou estranho, pois o nome da função é `obter_limite`, mas a variável com o valor a ser retornado se chama `valor_credito_disponivel`. Meio confuso isso. Estamos aí calculando o que?

O objetivo da função é retornar o limite de gastos.